$NetBSD: patch-config_system-headers,v 1.19 2016/06/16 12:08:21 ryoon Exp $

* Try to support Video4Linux2
* Resolve libm undefined error
* Support system libraries

--- config/system-headers.orig	2016-06-01 04:11:40.000000000 +0000
+++ config/system-headers
@@ -631,6 +631,7 @@ libgnome/libgnome.h
 libgnomeui/gnome-icon-lookup.h
 libgnomeui/gnome-icon-theme.h
 libgnomeui/gnome-ui-init.h
+libv4l2.h
 limits
 limits.h
 link.h
@@ -724,6 +725,7 @@ mapiutil.h
 mapix.h
 Math64.h
 math.h
+complex
 mbstring.h
 #ifdef ANDROID
 media/AudioEffect.h
@@ -1315,6 +1317,25 @@ vpx/vp8cx.h
 vpx/vp8dx.h
 vpx_mem/vpx_mem.h
 #endif
+#ifdef GKMEDIAS_SHARED_LIBRARY
+vpx/vpx_codec.h
+vpx/vpx_decoder.h
+vpx/vpx_encoder.h
+vpx/vp8cx.h
+vpx/vp8dx.h
+vpx_mem/vpx_mem.h
+vorbis/codec.h
+theora/theoradec.h
+tremor/ivorbiscodec.h
+speex/speex_resampler.h
+soundtouch/SoundTouch.h
+ogg/ogg.h
+ogg/os_types.h
+nestegg/nestegg.h
+cubeb/cubeb.h
+speex/speex_resampler.h
+soundtouch/SoundTouch.h
+#endif
 gst/gst.h
 gst/app/gstappsink.h
 gst/app/gstappsrc.h
@@ -1345,3 +1366,35 @@ unicode/utypes.h
 #endif
 libutil.h
 unwind.h
+#if MOZ_NATIVE_OGG==1
+ogg/ogg.h
+ogg/os_types.h
+#endif
+#if MOZ_NATIVE_THEORA==1
+theora/theoradec.h
+#endif
+#if MOZ_NATIVE_VORBIS==1
+vorbis/codec.h
+vorbis/vorbisenc.h
+#endif
+#if MOZ_NATIVE_TREMOR==1
+tremor/ivorbiscodec.h
+#endif
+#if MOZ_NATIVE_CELT==1
+celt.h
+celt_header.h
+celt/celt.h
+celt/celt_header.h
+#endif
+#if MOZ_NATIVE_OPUS==1
+opus.h
+opus_multistream.h
+opus/opus.h
+opus/opus_multistream.h
+#endif
+#if MOZ_NATIVE_SPEEX==1
+speex/speex_resampler.h
+#endif
+#if MOZ_NATIVE_SOUNDTOUCH==1
+soundtouch/SoundTouch.h
+#endif
